### PR TITLE
Code new - Banners and few changes

### PR DESCRIPTION
**Pull request by SomeOrdinaryBro**

### CODE REVIEW

**Comment by SomeOrdinaryBro** [comment]
will add more, pls dont merge unless u have anything urgent to push

**Comment by SomeOrdinaryBro** [comment]
**Added some clarity to the section you added. please review and make sure its accurate before meging**

- (Line 260) there is a commented block, check if its appropriate to add to this tutorial. I was doing some research and came across it.

File: Before You Start/4. Git & GitHub Mastery.md [hunk 1]
```diff
@@ -110,98 +110,163 @@ For this tutorial, we will install the standalone installer and explain how to i
...
+
+Think of it like a time machine for your work. Every time you make a change or save something new, Git takes a snapshot and stores it in this control room. So, if you ever need to go back in time to see how things looked before, Git has got you covered.
+
+**Question: What if I already have an existing project that I want to add to Git?**
+
+If you already have a project and want to start using Git, it's as easy as running `git init` in your project's folder. This action activates Git's superpower for your existing work, allowing you to keep track of changes, collaborate with others, and explore the history of your project.
+
+
+<!-- Here are some additional things to keep in mind when creating a Git repository:
```
Comment (SomeOrdinaryBro) [inline Before You Start/4. Git & GitHub Mastery.md:260]: This Block. Please Review

**Comment by SomeOrdinaryBro** [comment]
Merge if you want, I have pushed everything I changed/added.

**Comment by mshajid** [comment]
Alright. let me merge this commits for now. @SomeOrdinaryBro

**Pull request merged**